ci/performance_tests/test_performance.py

```diff
@@ -138,7 +138,7 @@ def warm_up_cluster(num_nodes, object_store_memory):
         for i in range(num_nodes):
             for _ in range(num_objects):
                 object_ids += [
-                    create_array._remote(args=[size], resources={str(i): 1})
+                    create_array.options(args=[size], resources={str(i): 1})
```

Comment:
```suggestion
                    create_array.options(resources={str(i): 1}).remote(size)
```
don't we need to call `.remote()` here?